### PR TITLE
Z->z

### PR DESCRIPTION
**Pull request by XintongLi2023**

### CODE REVIEW

File: R/pvalue_maxcombo.R [hunk 1]
```diff
@@ -61,13 +61,13 @@ NULL
...
 #'
-pvalue_maxcombo<- function(Z,
+pvalue_maxcombo<- function(z,
                            dummy_var,
                            algorithm = GenzBretz(maxpts = 50000, abseps = 0.00001)){
 
-  ans <- (1 - mvtnorm::pmvnorm(lower = rep(Z$Z %>% min() %>% as.numeric(),
-                                           nrow(Z)),
-                               corr = Z %>%
+  ans <- (1 - mvtnorm::pmvnorm(lower = rep(z$Z %>% min() %>% as.numeric(),
```
Comment (LittleBeannie) [inline R/pvalue_maxcombo.R:68]: Could you please also change the `$Z` into `$z`? This `$Z` is an output variable from `sim_fixed_n`. So please also remember to update the `Z` in `sim_fixed_n`.

**Review by LittleBeannie — COMMENTED** [review]
Hi @XintongLi2023, here is one more `Z` to be updated.

File: R/pvalue_maxcombo.R (hunk 1 above)
Comment (XintongLi2023) [reply R/pvalue_maxcombo.R]: Revised and submit it again. Please check my revisions.

**Review by LittleBeannie — APPROVED** [review]
Thanks for taking the time to overcome this challenging ticket. All I can say is THANK YOU!

**Pull request merged**